glommio/src/io/dma_file_stream.rs

```diff
@@ -698,7 +698,7 @@ impl AsyncRead for DmaStreamReader {
 pub struct DmaStreamWriterBuilder {
     buffer_size: usize,
     write_behind: usize,
-    flush_on_close: bool,
+    sync_on_close: bool,
```

Comment:
already what it is called in public api `with_sync_on_close_disabled()`

Reply:
yeah, those changes are good.